packages/elements-core/src/components/TableOfContents/constants.ts

```diff
@@ -20,19 +21,19 @@ export const NODE_TYPE_META_ICON: { [nodeType: string]: IIconProps['icon'] } = {
   model: faCube,
 };
 
-export const NODE_TYPE_ICON_COLOR = {
+export const NODE_TYPE_ICON_COLOR: Partial<Dictionary<string, NodeType>> = {
```

Comment:
lets make these `readonly`

Reply:
Done in dc54631ee990f5b015634c93e8449af47b27c366